plaid/processor.go

```diff
@@ -5,26 +5,14 @@ import (
 	"errors"
 )
 
-type createApexTokenRequest struct {
+type createProcessorTokenRequest struct {
```

Comment:
this will be a breaking change for apex + dwolla integrations – can we copy-pasta this for ocrolus and leave the existing request structs?

Reply:
Actually – since the `create...Request` is never exported we can make this generic.

The `Create...Response` is exported however, so we need to keep those

Reply:
disregard – tests are passing and is not a breaking change

Reply:
Good point. Since we're exporting the `Create...Response` types, we should probably keep those around. I've fixed that in commit e07e7cf.